lib/ansible/module_utils/network/ios/config/lldp_interfaces/lldp_interfaces.py

```diff
@@ -0,0 +1,267 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The ios_lldp_interfaces class
+It is in this file where the current configuration (as dict)
+is compared to the provided configuration (as dict) and the command set
+necessary to bring the current configuration to it's desired end-state is
```

Comment:
same typo, please check it and change to '...configuration to its desired...'

Reply:
@danielmellado I didn't get this comment, u meant to change following `current configuration to it's desired` to ?, also since it was rmb generated didn't look much into these :D

Reply:
I mean that it should be written `current configuration to its desired' , removing it's. IMHO that's a typo

Reply:
updated